### DISCUSSION TITLE
Low-Bitrate Switcher

### DISCUSSION THREAD

**Discussion by DJDevil2K** [datarhei/restreamer#805] [Ideas]
First of all... thank you for Restreamer ! I would like to know if there is something planned like a low-bitrate switcher or "be back soon" option if bitrate drops below a chosen value. Would be a cool option for people who dont use OBS. I've installed my Restreamer on VPS, make a SRT Server and Streaming with my mobile and HEVC, almost the perfect solution.

**Comment by nils-kt** [comment]
Any news about this?
